public/assets/css/modules/button--generate-style-guide.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  background-color: #C42872;
+  color: #fff;
+  height: 113px;
+  position: absolute;
+  right: -56px;
+  top: 300px;
+  transform: rotateZ(45deg) rotateX(0deg) rotateY(180deg);
+  z-index: 10;
+  width: 113px;
+}
+
+#text-button {
```

Comment:
What do you think about changing this to be a CSS class instead of an ID, or is there a reason you need this to be an ID, like JS? Tell me more about it.